### ISSUE TITLE
"H.265/HEVC Video Decoder Filter" option in Detailed Setup

### ISSUE THREAD

**Issue by 49studebaker** [google/sagetv#93]
Add "H.265/HEVC Video Decoder Filter" option to "Video/Audio" in "Detailed Setup".

**Comment by CraziFuzzy** [comment]
Expanding on this, I think far more can be done for setting format codecs than just adding HEVC.  Audio options for instance can be broken down as well.  (AAC, AC3, DTS, etc).

**Comment by Narflex** [comment]
It already does that via properties internally, so these are really just
STV changes.

Jeff Kardatzke

**Comment by CraziFuzzy** [comment]
Yeah, I know the properties are there, but I've notices some strange behaviors - I think there are some things hard-coded in the player (or perhaps it was more in the video rendering that I've seen that).  I do know people would love to use MadVR from the SageTVClient, but the renderer choices are hard-coded.
